ecs-agent/tmds/handlers/fault/v1/types/types.go

```diff
@@ -80,6 +80,34 @@ func (response NetworkFaultInjectionResponse) ToString() string {
 	return string(data)
 }
 
+type NetworkLatencyRequest struct {
+	DelayMilliseconds  *uint64   `json:"DelayMilliseconds"`
+	JitterMilliseconds *uint64   `json:"JitterMilliseconds"`
+	Sources            []*string `json:"Sources"`
```

Comment:
I think it'd be nice to have a comment here explaining what the format of the `Sources` strings is assumed to be.

Reply:
Sure will do

Reply:
Added